h5py/h5f.pyx

```diff
@@ -582,3 +583,42 @@ cdef class FileID(GroupID):
             Feature requires: 1.9.178 HDF5
             """
             H5Fstart_swmr_write(self.id)
+
+    IF HDF5_VERSION >= (1, 10, 1):
+
+        @with_phil
+        def reset_page_buffering_stats(self):
+            """ ()
+
+            Reset page buffer statistics for the file.
+            """
+            H5Freset_page_buffering_stats(self.id)
+
+        @with_phil
+        def get_page_buffering_stats(self):
+            """ () -> PageBufStats(MetaStats, RawStats)
+
+            Retrieve page buffering statistics for the file as the number of
+            metadata and raw data accesses, hits, misses, evictions, and
+            accesses that bypass the page buffer (bypasses).
+            """
+            cdef:
+                unsigned int accesses[2]
+                unsigned int hits[2]
+                unsigned int misses[2]
+                unsigned int evictions[2]
+                unsigned int bypasses[2]
+
+            H5Fget_page_buffering_stats(
+                self.id, &accesses[0], &hits[0], &misses[0], &evictions[0], &bypasses[0])
+            PageBufStats = namedtuple('PageBufferStats', ['meta', 'raw'])
+            MetaStats = namedtuple(
+                'MetaStats', ['accesses', 'hits', 'misses', 'evictions', 'bypasses'])
+            RawStats = namedtuple(
+                'RawStats', ['accesses', 'hits', 'misses', 'evictions', 'bypasses'])
```

Comment:
Can these be moved to the top-level scope? It would be cleaner and avoid defining them on each call.

Reply:
Sure, done in #d6a868f.